assets/css/main.css

```diff
@@ -2372,37 +2372,18 @@
         position: relative;
     }
 
-        .icon:before {
-            -moz-osx-font-smoothing: grayscale;
-            -webkit-font-smoothing: antialiased;
-            font-family: FontAwesome;
+        .icon::before {
+            display: inline-block;
             font-style: normal;
-            font-weight: normal;
-            text-transform: none !important;
+            font-variant: normal;
+            text-rendering: auto;
+            -webkit-font-smoothing: antialiased;
         }
 
         .icon > .label {
             display: none;
         }
 
-        .icon.alt {
```

Comment:
I don't think this should have been removed, I could be wrong though.

Reply:
What is it used for?
All I found it did was break the fontawesome rendering and attempt to add a "solid" fill to line-fill icons. I used the fontawesome 5 solid icon set.

Reply:
Perhaps V5 no longer requires alt icons in cases of different reses